src/components/organisms/Badges/Badges.tsx

```diff
@@ -84,29 +115,29 @@ export const Badges: React.FC<{
 
         return {
           venue,
-          image: venue.mapIconImageUrl,
+          image: venue?.host?.icon,
           label: venue.name,
         };
       })
       .filter((b) => b !== undefined);
-  }, [relevantVisits, venues, currentVenue, relatedVenueNames]);
+  }, [relevantVisits, venues]);
 
   const badgeList = useMemo(
     () =>
-      badges.filter(notEmpty).map((b) => (
-        <li className="badge-list-item" key={b.label}>
-          <Link to={getLocationLink(b.venue, b.room)}>
-            <img
-              className="badge-list-item-image"
-              src={b.image}
-              alt={`${b.label} badge`}
-            />
+      badges.filter(notEmpty).map((badge) => (
+        <li className="badge-list-item" key={badge.label}>
+          <Link to={getLocationLink(badge.venue, badge.room)}>
+            <BadgeImage image={badge.image} />
```

Comment:
We should restore the passing through of the venue title here so we can use it in the badge's alt text. 

Something like (this may not be accurate as is, treat it as pseudocode):
```suggestion
            <BadgeImage image={badge.image} badgeName={badge.venue.name} />
```